docs/js/modules/uiElements.js

```diff
@@ -77,14 +77,14 @@ function modal(id){
    }
 }
 
-export function connectionErrorToast(error, defaultErrorText = 'we couldn\'t get the latest data.'){
-   let toastHTML = '';
-   //simple connection error
-   if(error.status === 0 || error.statusText === ""){
-      toastHTML = '<p>We couldn\'t get the latest data. Please check your connection</p>';
-   }//more serious problem
-   else{
-      toastHTML = `<p>${defaultErrorText} Error ${error.status}</p>`;   
-   }
-   M.toast({html: toastHTML,displayLength:2000, classes:"dark-toast"}); 
-}
+// export function connectionErrorToast(error, defaultErrorText = 'we couldn\'t get the latest data.'){
```

Comment:
This removes the error message that pops up saying that it can't connect to the API. There's no reason to warn people that it doesn't connect correctly since we are temporarily removing API-related features.

Reply:
Maybe we should remove the code that polls the apis for updates instead of removing the warnings. We only have to comment out the inits from the main iife, without having to remove important classes